### ISSUE TITLE
[User flows and navigating] Link or button is not working as expected or did not navigate to the intended destination. (03.08.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#50804]
#### VFS team name
Sitewide Content

#### VFS product name
Modernized accessibility page

#### Point of Contact/Reviewers
Sara Sterkenburg - @sterkenburgsara - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Link or button is not working as expected or did not navigate to the intended destination.

### Issue Details
According to the IA review document submitted by your team (Joni), the accessibility link in the footer should have the url: va.gov/accessibility-at-va rather than the older 508 compliance link.

### VA.gov Experience Standard
[Category Number 03, Issue Number 08](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
Replace footer link with link to this new page va.gov/accessibility-at-va

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.
